modyn/selector/internal/selector_strategies/abstract_selection_strategy.py

```diff
@@ -391,3 +391,12 @@ def _persist_samples(self, keys: list[int], timestamps: list[int], labels: list[
 
         for proc in processes:
             proc.join()
+
+    def seed_selector(self, seed: float) -> bool:
+        with MetadataDatabaseConnection(self._modyn_config) as database:
+            if database.drivername == "postgresql":
+                assert -1 <= seed <= 1
```

Comment:
The problem is that all libraries want an integer while postgres wants a float between [-1,+1]. Since having two different seeds doesn't seem necessary, only the _int_ seed is requested in the pipeline and the float is taken from there (rescaling the value between [-1,1]). Not the most elegant solution but it ensures reproducibility